lib/ui/tutorials/tutorial_sliders.dart

```diff
@@ -0,0 +1,298 @@
+import 'package:flutter/services.dart';
+import 'package:geolocator/geolocator.dart';
+import 'package:intro_slider/intro_slider.dart';
+import 'package:intro_slider/slide_object.dart';
+
+import '../../ui_imports.dart';
+
+class TutorialSliders extends StatefulWidget {
+  @override
+  _TutorialSlidersState createState() => _TutorialSlidersState();
+}
+
+class _TutorialSlidersState extends State<TutorialSliders> {
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  void onDonePress() {
+    // Do what you want
+  }
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
Could we split this change?
A) Just some intro tutorial pages which we discuss also with e.g. helmi
B) The interactive map tutorial
C) Location permission in the tutorial?